app/controllers/api/channels_controller.rb

```diff
@@ -54,21 +57,41 @@ def verify
   end
 
   def show
-    render(json: @channel.details)
+    render(json: @channel.details, status: :ok)
+  end
+
+  def create
+    channel = Channel.new(publisher: @owner, verified: true)
+
+    channel.details = SiteChannelDetails.new(site_channel_details_params)
```

Comment:
why are we only supporting site channels? what's the context for this endpoint / when do we see ourselves using it?

Reply:
i have been waiting to use it since before the transition for one multi-site publisher (-;